mod/profile.php

```diff
@@ -181,6 +181,10 @@ function profile_content(&$a, $update = 0) {
 		$commpage = (($a->profile['page-flags'] == PAGE_COMMUNITY) ? true : false);
 		$commvisitor = (($commpage && $remote_contact == true) ? true : false);
 
+		if(feature_enabled($a->profile['profile_uid'],'photos_widget')) {
+			require_once('include/photos.php');
```

Comment:
In php5 should not have any influence. The first time `require_once()` is called, the content of the file is added to the global context, so there is no difference in calling it in the middle of the code or on top (I think. I've done some tests and looks it works like this).
Having all `require_once()` on top of the file make easier track file dependencies, IMHO.
It's a "coding style" thing :-)

Well, in theory  if isn't needed, isn't loaded, so it should be better like that, but is "ugly". The correct solution would be convert the code to use autoloader...